tasks/src/image-to-text/data.ts

```diff
@@ -0,0 +1,58 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Dataset from 12M image-text of Reddit",
+			id:          "red_caps",
+		},
+        {
+			// TODO write proper description
+			description: "Dataset from images generated by Stable diffusion on Discord",
+			id:          "poloclub/diffusiondb",
+		},
+
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "savanna.jpg",
+				type:    "img",
+			},
+		],
+		outputs: [
+			{
+				label:   "Detailed description",
+				content: "a herd of giraffes and zebras grazing in a field",
+				type:    "text",
+			},
+		],
+	},
+	metrics: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "",
+		}
+	],
+	models: [
+        {
+            description: "Donut consists of a vision encoder (Swin Transformer) and a text decoder (BART)",
+            id:          "naver-clova-ix/donut-base-finetuned-docvqa",
+        },
+		{
+			description: "This is an image captioning model training",
+			id:          "nlpconnect/vit-gpt2-image-captioning",
+		},
+	],
+	summary:      "Image classification is the task of assigning a label or class to an entire image. Images are expected to have only one class for each image. Image classification models take an image as input and return a prediction about which class the image belongs to.",
```

Comment:
```suggestion
	summary:      "",
```

I think you copied and pasted the description from somewhere🙂